docs/manual/user_guide.adoc

```diff
@@ -131,10 +131,10 @@ Putting these arguments together, a properly formatted command would be:
 
 ----
 $ sudo oscap xccdf eval --profile stig-rhel6-server \
---results /root/ssg-results.xml \
---report /root/ssg-report.xml \
---cpe /usr/share/scap/ssg/ssg-rhel6-cpe-dictionary.xml \
-/usr/share/scap/ssg/ssg-rhel6-xccdf.xml
+--results /tmp/results.xml \
+--report /tmp/report.xml \
```

Comment:
It is better if a report has `html` extension.

Reply:
Agreed.